docker-compose.yml

```diff
@@ -199,16 +199,19 @@ services:
       - generated-env/env.io-backend
     labels:
       - "traefik.enable=true"
+      - "traefik.docker.network=io-fn"
```

Comment:
is there any offline configuration needed to make traefik get the rights TSL certs or it's everything automatic?

Reply:
Automatic, no offline configuration is needed